src/pages/home/Home.tsx

```diff
@@ -51,6 +52,7 @@ const Home = () => {
                 type="text"
                 className="searchInput"
                 placeholder="What do you want to watch?"
+                onChange={(e) => search(e.target.value)}
```

Comment:
You can create a separate function, `onSearch` for example and then just pass function reference to `onChange`